evap/staff/importers.py

```diff
@@ -574,55 +574,45 @@ def write_enrollments_to_db(self, semester, vote_start_datetime, vote_end_date):
-            count=responsible_word
-        )
-        evaluation_phrase = ngettext(
-            "{count} course/evaluation", "{count} courses/evaluations", evaluation_count
-        ).format(count=evaluation_word)
+        if not students_created and not responsibles_created and not self.evaluations:
+            return _("Nothing changed")
+
+        student_phrase = self.make_phrase(len(students_created), "student", "students")
+        responsible_phrase = self.make_phrase(len(responsibles_created), "responsible", "responsibles")
+        evaluation_phrase = self.make_phrase(len(self.evaluations), "course/evaluation", "courses/evaluations")
 
         message = _("Successfully created {evaluation_phrase}, {student_phrase} and {responsible_phrase}").format(
             responsible_phrase=responsible_phrase, student_phrase=student_phrase, evaluation_phrase=evaluation_phrase
         )
-        if student_count == 0 and responsible_count == 0 and evaluation_count == 0:
-            message = "Nothing changend"
-        elif student_count > 0 or responsible_count > 0:
+
+        if students_created or responsibles_created:
             message += ":"
             message += create_user_list_html_string_for_message(students_created)
             message += create_user_list_html_string_for_message(responsibles_created)
         self.success_messages.append(message)
+        return message
 
     def create_test_success_messages(self):
         filtered_users = [user_data for user_data in self.users.values() if not user_data.user_already_exists()]
 
         self.success_messages.append(_("The test run showed no errors. No data was imported yet."))
 
-        user_count = len(filtered_users)
-        user_word = pgettext("count", "no") if user_count == 0 else str(user_count)
-        user_phrase = ngettext("{count} user", "{count} users", user_count).format(count=user_word)
-        evaluation_count = len(self.evaluations)
-        evaluation_word = pgettext("count", "no") if evaluation_count == 0 else str(evaluation_count)
-        evaluation_phrase = ngettext(
-            "{count} course/evaluation", "{count} courses/evaluations", evaluation_count
-        ).format(count=evaluation_word)
+        user_phrase = self.make_phrase(len(filtered_users), "user", "users")
+        evaluation_phrase = self.make_phrase(len(self.evaluations), "course/evaluation", "courses/evaluations")
 
         msg = _("The import run will create {evaluation_phrase} and {user_phrase}").format(
             evaluation_phrase=evaluation_phrase, user_phrase=user_phrase
         )
-        if user_count == 0:
-            msg += format_html(_("."))
+        if not filtered_users:
+            msg += "."
         else:
-            msg += format_html(_(":"))
-            msg += create_user_list_html_string_for_message(filtered_users)
+            msg += format_html("{}: {}", msg, create_user_list_html_string_for_message(filtered_users))
```

Comment:
Same thing here: This does `str + SafeString`, you probably meant to do `=` and not `+=`

Reply:
Fixed the +=. I will leave this open as a reminder for you two to figure out your differences ;)